src/Parsers/ASTFunction.cpp

```diff
@@ -464,6 +464,8 @@ ASTPtr ASTFunction::clone() const
         res->children.push_back(res->window_definition);
     }
 
+    res->prefer_subquery_to_function_formatting = prefer_subquery_to_function_formatting;
```

Comment:
as I understand you do not need to copy this value manually because it will be copied in `auto res = std::make_shared<ASTFunction>(*this);` because this value is just a bool.

Reply:
oh you are right, removed this and other assigned variables from clone function.